### PR TITLE
Replace App model with Integration model

### PR DESCRIPTION
**Pull request by orenyk**
Replace App model with Integration model and set up dashboard. This
includes:
- Enum field for protocol backed by a string column
- Boolean field for IAL2 (assume all integrations will be IAL1)
- Ignore estimates and usage tracking for now

Not sure this is done but wanted to get it ready. Ideally I'd like to figure out how to get the protocol acronyms displayed in all caps before we merge in but it's not strictly necessary.

Blocked on #7

### CODE REVIEW

File: db/migrate/20201104021414_create_integrations.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
+class CreateIntegrations < ActiveRecord::Migration[6.0]
+  def change
+    create_table :integrations do |t|
+      t.string :issuer, null: false
+      t.string :name, null: false
+      t.text :description
+      t.boolean :ial2, null: false, default: false
+      t.string :protocol, null: false, default: 'oidc'
+      t.string :url
+      t.date :go_live
```
Comment (orenyk) [inline db/migrate/20201104021414_create_integrations.rb:10]: Add separate production deploy date attribute

Comment (orenyk) [reply db/migrate/20201104021414_create_integrations.rb:10]: Also add dashboard URL

Comment (orenyk) [reply db/migrate/20201104021414_create_integrations.rb:10]: Make sure to add them to the dashboard

File: config/initializers/inflections.rb [hunk 2]
```diff
@@ -3,5 +3,7 @@
   inflect.acronym('GTCs')
   inflect.acronym('IAA')
   inflect.acronym('IAAs')
+  inflect.acronym('OIDC')
```
Comment (orenyk) [inline config/initializers/inflections.rb:6]: See if you can figure out how to display these as uppercase in Administrate

**Review by achapm — APPROVED** [review]
LGTM

**Pull request merged**